.azure-pipelines/code-quality.yml

```diff
@@ -14,6 +14,19 @@ steps:
         Rscript './.azure-pipelines/scripts/validate_copyright_header.R' 'R'
     displayName: 'Validate copyright header'
 
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: |
+        pip install --user azureml-sdk
+        R -e 'dir.create(Sys.getenv("R_LIBS_USER"), recursive = TRUE);
+              .libPaths(Sys.getenv("R_LIBS_USER"));
+              # build and install R sdk
+              devtools::install_deps(upgrade = FALSE);
+              package_location <- devtools::build();
+              install.packages(package_location, repos = NULL)'
+    displayName: 'Build and Install SDK'
```

Comment:
Still need to install SDK? What happened?

Reply:
because both R CMD check and lintr would fail if they don't find azureml package.